packages/gatsby-source-wordpress/src/__tests__/normalize.js

```diff
@@ -57,6 +57,10 @@ describe(`Process WordPress data`, () => {
       { id: 2, acf: {} },
     ])
   })
+  it(`Creates links between entities and polylang translations entities`, () => {
+    entities = normalize.mapPolylangTranlations(entities)
```

Comment:
I don't think it test anythign given that entities here don't have `polylang_translations` field

Reply:
@pieh Sorry, it's the first time I'm doing testing.

I've add `Polylang_translations` in `data.json` and change the snapshot. I've tried to inspired myself with the other test who link two entities together.